problems/1.types+operators.js

```diff
@@ -3,15 +3,19 @@
 * @returns {boolean} - Is val a number
 */
 function isNumber() {
-
+let val = 1
+  return val
 }
 
+
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val) {
+ 
+  return true 
```

Comment:
This only returns true. What if the val that was passed was false ? 
We need to check for truthyness. Remember the falsey values in JS are  0, "",  null, undefined, NaN, false. 
Use a condition or !!